### ISSUE TITLE
CertStreamEventStream panics on nil interfaces

### ISSUE THREAD

**Issue by zuBux** [CaliDog/certstream-go#6]
CertStreamEventStream crashes on the following line:

```
panic: interface conversion: interface {} is nil, not map[string]interface {}

goroutine 81 [running]:
github.com/CaliDog/certstream-go.CertStreamEventStream.func1(0xc420458180, 0xc420458120, 0x0)
        /home/ubuntu/go/src/github.com/CaliDog/certstream-go/certstream.go:34 +0x33b
created by github.com/CaliDog/certstream-go.CertStreamEventStream
        /home/ubuntu/go/src/github.com/CaliDog/certstream-go/certstream.go:48 +0x97
```

The root cause is [this](https://github.com/jmoiron/jsonq/blob/master/jsonq.go#L85). Maybe an additional check should be added?

**Comment by Fitblip** [comment]
Hey @zuBux,

Thanks for the bug report.

> The root cause is this. Maybe an additional check should be added?

Hmm, so what do you think the additional check should be? One big beef I have with Go's empty interfaces is the API is very confusing when it comes to checking for if an empty interface is null 😓.

**Comment by VegarLH** [comment]
Does pull request #7 work for you @zuBux ?

**Comment by zuBux** [comment]
Hey, 

Thanks for this! I'll test it today and let you know :)

**Comment by Fitblip** [comment]
Just to update this, we're hammering out some small details here and will shortly have this fixed :)

**Comment by Fitblip** [comment]
Hey @zuBux this should be fixed in #7 - thanks to @tcpepidemic for the fix!

**Comment by zuBux** [comment]
Thx!
